doc/documentation_testing.md

```diff
@@ -0,0 +1,38 @@
+Testing Python examples in the documentation
+============================================
+We can use the Python doctest module for testing examples in the documentation.
+Run it with
+
+    python -m doctest <filename>
+
+on your markdown file.
+
+See [documentation contributions] for how to write your examples such that they can be tested with doctest.
+
+For instance, assume that you have the following example in your documentation
+
+    ```python
+        >>> 1 + 1  # doctest: +SKIP
+        3
+
+    ```
```

Comment:
```suggestion
```python
>>> 1 + 1  # doctest: +SKIP
3
```
```

Reply:
Doesn't work. The closing inner backticks are stripped off...